source/plugins/go/src/telemetry.go

```diff
@@ -42,6 +42,10 @@ var (
 	ContainerLogsSendErrorsToMDSDFromFluent float64
 	//Tracks the number of mdsd client create errors for containerlogs (uses ContainerLogTelemetryTicker)
 	ContainerLogsMDSDClientCreateErrors float64
+	//Tracks the number of mdsd client create errors for insightsmetrics (uses ContainerLogTelemetryTicker)
+	InsightsMetricsMDSDClientCreateErrors float64
+	//Tracks the number of mdsd client create errors for kubemonevents (uses ContainerLogTelemetryTicker)
+	KubeMonEventsMDSDClientCreateErrors float64
```

Comment:
where are these telemetry flushed ?

Reply:
thanks for catching and it was miss. fixed